examples/lang/erc20/src/lib.rs

```diff
@@ -152,12 +152,23 @@ contract! {
 mod tests {
     use super::*;
     use ink_core::env;
-    use std::convert::TryFrom;
+    use parity_codec::Decode;
+
+    /// Create a new AccountId with the given bytes
+    ///
+    /// Required because AccountId is a type alias and cannot be used as a constructor
+    fn new_account_id(data: [u8; 32]) -> AccountId {
```

Comment:
Maybe in this case trait bounds `TryFrom<&[u8]>` or `From<[u8;32]>` would be useful for `AccountId` and maybe some other types.

Reply:
We should really not force people to write functions like these for their simple test codes.